src/pyhf/tensor/jax_backend.py

```diff
@@ -57,6 +57,13 @@ def _setup(self):
         Run any global setups for the jax lib.
         """
 
+    @property
+    def tensor(self):
+        """
+        Provide the underlying tensor type.
+        """
+        return np.array
```

Comment:
I think this should be
```suggestion
        return jax.interpreters.xla.DeviceArray
```

given

```python
>>> import pyhf
>>> pyhf.set_backend("numpy")
>>> x = pyhf.tensorlib.astensor([10.])
>>> print(f"x is a {type(x)}")
x is a <class 'numpy.ndarray'>
>>> pyhf.set_backend("jax")
>>> y = pyhf.tensorlib.astensor([10.])
>>> print(f"y is a {type(y)}")
y is a <class 'jax.interpreters.xla.DeviceArray'>
```